lib/services.identity.js

```diff
@@ -516,11 +516,6 @@ api._createIdentity = function(options, req, callback) {
     }
     // result details
     var details = {identity: results.createIdentity};
-    // schedule identity created event
-    bedrock.events.emitLater({
-      type: 'bedrock.Identity.created',
```

Comment:
This doesn't get used anywhere, and there is a postInsert event that gets generated from brIdentity itself, so I thought we'd remove it and just use the postInsert event.